src/components/Layout/Header/NavBar/KeepKey/LastDeviceInteractionStatus.tsx

```diff
@@ -0,0 +1,37 @@
+import { Alert, AlertIcon, useColorModeValue } from '@chakra-ui/react'
+import { upperFirst } from 'lodash'
+import { useTranslate } from 'react-polyglot'
+import { useWallet } from 'hooks/useWallet/useWallet'
+
+export type LastDeviceInteractionStatusProps = {
+  setting: string
+}
+
+export const LastDeviceInteractionStatus = ({ setting }: LastDeviceInteractionStatusProps) => {
+  const translate = useTranslate()
+  const {
+    state: { lastDeviceInteractionStatus }
+  } = useWallet()
+  const greenShade = useColorModeValue('green.700', 'green.200')
+  const yellowShade = useColorModeValue('yellow.500', 'yellow.200')
+
+  return lastDeviceInteractionStatus ? (
+    <Alert
+      status={lastDeviceInteractionStatus === 'success' ? 'success' : 'error'}
```

Comment:
Nitpick: If we make `Outcome` to be `export type Outcome = 'success' | 'error' | undefined`, you could directly use it here instead of a ternary, since you won't reach this branch in case it's undefined.

Reply:
I like it - updated in https://github.com/shapeshift/web/pull/1341/commits/284f1ac95af39579135508ea712026483fb6cbc1